.github/workflows/build-gem.yml

```diff
@@ -0,0 +1,21 @@
+name: Check if Gem exists in production
+
+on: [pull_request]
+
+jobs:
+  build-gem:
+    runs-on: ubuntu-latest
+    continue-on-error: true
+    
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Ruby
+        uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: 2.7.6
+      - name: Check if gem exists
+        run: |
+          GEM_VERSION=$(ruby -e "require 'scc/codestyle'; print Scc::Codestyle::VERSION" -I lib)
+          # if we install the gem, then the $GEM_VERSION already exist in production
+          # therefore error
+          gem install -q --silent scc-codestyle -v $GEM_VERSION || exit 0
```

Comment:
Is it possible to fail with an error message like, please bump the gem version, the current version is already released on rubygems.org?

Reply:
I'm having issue to do it since github actions wrap each command in a  `bash -e`  call which exits when it errors.. any idea how to make it work?